frontend/src/core/comments/components/CommentsList.css

```diff
@@ -18,9 +18,3 @@
     width: 32px;
     height: 32px;
 }
-
-.comments-list .comment .container {
-    display: flex;
-    flex-flow: column wrap;
-    width: 100%;
-}
```

Comment:
Why only move this CSS rule to `Comment.css`, but not the other `.comments-list .comment` rules?

Also, please remove the `margin-left: 4px;` from the `.comments-list .comment .content p` rule in the `Comment.css` file, because it's no longer needed and makes passing inconsistent between submitted comments and the add comment form.

Reply:
> Why only move this CSS rule to `Comment.css`, but not the other `.comments-list .comment` rules?
> 
> Also, please remove the `margin-left: 4px;` from the `.comments-list .comment .content p` rule in the `Comment.css` file, because it's no longer needed and makes passing inconsistent between submitted comments and the add comment form.

I only moved this one because it was the only one no longer being shared by both comment and add comment.  However, the latest changes reverses this and they now share most of these styles and so I moved it back.